packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -2704,7 +2714,7 @@ class RectangularRangeSliderValueIndicatorShape
       textScaleFactor: textScaleFactor!,
       sizeWithOverflow: sizeWithOverflow!,
       backgroundPaintColor: sliderTheme!.valueIndicatorColor!,
-      strokePaintColor: isOnTop! ? sliderTheme.overlappingShapeStrokeColor : null,
+      strokePaintColor: isOnTop! ? sliderTheme.overlappingShapeStrokeColor : sliderTheme.valueIndicatorStrokeColor,
```

Comment:
Looks like you added tests for all the value indicator shapes except  this `RectangularRangeSliderValueIndicatorShape`